blog/content/post/2023-02-24-msi_ms7d25_v1.1.1_release.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Dasharo compatible with MSI PRO Z690-A release v1.1.1
+abstract: 'A short overview of changes introduced in v1.1.1
+           release of Dasharo compatible with MSI PRO Z690-A
+	   with a short demo of Qubes OS 4.1.2.'
+cover: /covers/image-file.png
+author: michal.zygowski
+layout: post
+published: true
+date: 2023-02-24
+archives: "2023"
+
+tags:
+  - firmware
+  - coreboot
+  - Dasharo
+categories:
+  - Firmware
+
+
+---
+
+# Introduction
```

Comment:
Let's start with kudos to all who contributed to this release. Please use nicknames from github, forums and Dasharo community. Let's praise our community and contributors and thank them for hard work.

Reply:
https://github.com/3mdeb/news-and-ideas/pull/513/commits/0f29e7a980e1814bfdf6c3d8ab045c7ba6d63782#diff-b67e40bb2c6780be37b7e6cf6fc2a2a9f9ad977388795cc0c5bdfa6576dc52b6R24